outputs/html/index.ts

```diff
@@ -1,63 +1,27 @@
 // Copyright (c) Microsoft Corporation and others.
 // SPDX-License-Identifier: Apache-2.0
 
-import { LicenseBucket } from "../../structure";
-import OutputRenderer from "../base";
+import { LicenseBucket } from '../../structure';
+import OutputRenderer from '../base';
+import * as fs from 'fs';
+import * as path from 'path';
+import { compile } from 'handlebars';
+import { minify } from 'html-minifier';
 
 export default class HtmlRenderer implements OutputRenderer<string> {
-  private lines: string[] = [];
-
-  constructor() {}
+  constructor(private template?: string) {}
```

Comment:
@jpeddicord I see what you were saying before. This is not necessarily html now and should be renamed `TemplateRenderer`

Reply:
HtmlRenderer is fine to start with; I could see a separate template renderer extending this to work with multiple files/templates/etc.